main.py

```diff
@@ -0,0 +1,13 @@
+import sys
```

Comment:
Entry point for the docker file but also makes it easier to do `python main.py package -m test "v1.0.0" .`